hybridse/src/codegen/udf_ir_builder.cc

```diff
@@ -748,5 +752,47 @@ Status UdfIRBuilder::BuildUdafCall(
     return Status::OK();
 }
 
+Status UdfIRBuilder::BuildVariadicUdfCall(const node::VariadicUdfDefNode* fn,
+                                          const std::vector<const node::TypeNode*>& arg_types,
+                                          const std::vector<NativeValue>& args,
+                                          NativeValue* output) {
+    CHECK_TRUE(arg_types.size() == args.size(), kCodegenError);
+    CHECK_TRUE(fn->GetArgSize() == args.size(), kCodegenError);
+    std::vector<NativeValue> cur_state_values(1 + fn->update_func().size());
```

Comment:
it's a reduce algorithm ? So ideally can replace list of `NativeValue` into single accumulate `NativeValue` ?